### ISSUE TITLE
Would you accept additions from lewiscowper/dotcrap?

### ISSUE THREAD

**Issue by lewiscowper** [hauleth/xdg-rlz#2]
I came across this repo from a post on lobste.rs, and was thankful that someone else had the same thoughts [I had](https://lewiscowper.com/post/dotcrap/). I maintain a similar project to clear out "dotcrap" files into XDG specified directories, and I was wondering whether you'd be open to adding some of the entries [here](https://github.com/lewiscowper/dotcrap/blob/master/dotcrap) into your project, and collaborate on it in the future. Mine is predominantly (but not specifically) macOS/zsh based, but I see no reason exporting env variables would be shell or OS specific, other than the shell/OS specific files themselves.

**Comment by lewiscowper** [comment]
Further to this one, sometimes even with environment variables set some programs require flags to be set on the command as its run to specify alternative files (`tmux`, `kubectl`, `svn`, and `code` I think are the ones I've found). Perhaps adding aliases for these would either be useful to add in this project, or would be useful to document if the file is sourced.

**Comment by hauleth** [comment]
Yes, I am open to merging repos and even maybe host them as an organisation instead. I just added the entries for tools that I am using, but I am fully open to have it extended with more tools.
